LICENSE.md

```diff
@@ -1,7 +1,8 @@
-Written in 2017 by 
+Written in 2017-2022 by 
```

Comment:
_A little bikeshedding:_ is it a thing to have a different license than `http4s/http4s`?

Reply:
This would be a giter8 thing. They [advise using CC0](https://www.foundweekends.org/giter8/template.html#Use+CC0+1.0+for+template+licensing) for template user convenience.